### DISCUSSION TITLE
Clearing cookies and site data broken?

### DISCUSSION THREAD

**Discussion by BingLunsford** [mozilla/multi-account-containers#2679] [Q&A]
Clearing cookies and site data for a web site, from the option in the security icon address bar item, in one container appears to clear it in all other containers and the non-container.  As far as I understand it, it should not, right?  I mean, I don't know if this is a bug or a feature.

**Comment by kelimuttu** [comment]
Good point. Looking at #2686 and #2630, this might be a known issue.
